### PR TITLE
NY app regression failure fixes

### PR DESCRIPTION
**Pull request by kiran-tripathy**
## Summary
Fixing NY App script failures

### Link to Jira Ticket (if applicable)
[https://dimagi-dev.atlassian.net/browse/QA-4334](https://dimagi-dev.atlassian.net/browse/QA-4334)

## QA Checklist

- [x] Label(s) and Assignee added
- [x] PR sent for review
- [ ] Documentation (if applicable) added/updated

### CODE REVIEW

File: USH_Apps/NY_CICT/Utilities/user_inputs.robot [hunk 1]
```diff
@@ -9,4 +9,4 @@ ${pass}    %{DIMAGIQA_RF_PASSWORD}
 ${implcit_wait_time}    20
 ${app name}    NY
 ${archieved_contact_name}     Dewey
-${archieved_contact_lname}      SEPTEMBER13
\ No newline at end of file
+${archieved_contact_lname}      SEPTEMBER13
```
Comment (kbo001) [inline USH_Apps/NY_CICT/Utilities/user_inputs.robot:12]: Will these value remain the same everytime? Will the testcases pass?

File: USH_Apps/NY_CICT/requirements.txt [hunk 2]
```diff
@@ -5,3 +5,4 @@ robotframework-seleniumlibrary >= 3.3.1
 pyotp >= 2.6.0
 webdriver-manager
 robotframework-pabot >= 2.5.3
+robotframework-dependencylibrary >=3.0.1
```
Comment (kbo001) [inline USH_Apps/NY_CICT/requirements.txt:8]: good one

**Review by kbo001 — APPROVED** [review]
all looks good.

File: USH_Apps/NY_CICT/Utilities/user_inputs.robot (hunk 1 above)
Comment (kiran-tripathy) [reply USH_Apps/NY_CICT/Utilities/user_inputs.robot:12]: I've skipped these test cases, like we discussed last time. One has to create archived contacts manually and update the value here every time.

**Pull request merged**